srcpkgs/nilfs-utils/template

```diff
@@ -1,36 +1,38 @@
 # Template file for 'nilfs-utils'
 pkgname=nilfs-utils
-version=2.2.8
+version=2.2.9
 revision=1
 build_style=gnu-configure
-configure_args="--sbindir=/usr/bin --enable-libmount"
+configure_args="--without-selinux"
 make_install_args="sbindir=/usr/bin"
 hostmakedepends="automake libtool"
-makedepends="libuuid-devel libmount-devel"
+makedepends="libblkid-devel libmount-devel libuuid-devel"
 conf_files="/etc/nilfs_cleanerd.conf"
 short_desc="Log-structured file system for Linux - userspace utils"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-2.0-only, LGPL-2.1-only"
-homepage="http://nilfs.sourceforge.net/"
-distfiles="http://nilfs.sourceforge.net/download/$pkgname-$version.tar.bz2"
-checksum=5a29f8f59db03edf8dd1e68dfd80ff4713e0aa184cfa83a6326ac8af93d9bda6
+license="GPL-2.0-or-later, LGPL-2.1-or-later"
```

Comment:
Ah, OK. Grepping `COPYING` from source:

```
GNU GENERAL PUBLIC LICENSE

[…]

  9. The Free Software Foundation may publish revised and/or new versions
of the General Public License from time to time.  Such new versions will
be similar in spirit to the present version, but may differ in detail to
address new problems or concerns.

Each version is given a distinguishing version number.  If the Program
specifies a version number of this License which applies to it and "any
later version", you have the option of following the terms and conditions
either of that version or of any later version published by the Free
Software Foundation.  If the Program does not specify a version number of
this License, you may choose any version ever published by the Free Software
Foundation.
```

But this may not be the author’s intention, e.g. the source code states `Licensed under GPLv2`, `Licensed under LGPLv2` in many places.

I’ll change the license.

Reply:
The COPYING file must be copied verbatim from GNU, it's the requirement from GNU.
Part 2 of Clause 9 says 
```
If the Program
specifies a version number of this License which applies to it and "any
later version", you have the option of following the terms and conditions
either of that version or of any later version published by the Free
Software Foundation.
```
So, when source code doesn't mention "later version", it means that only version.

Reply:
IOW, the author is doing correctly (not violate GNU license), and licenses their software correctly under `GPL-2.0-only`